examples/Esp32-lwmqtt/esp32-mqtt.h

```diff
@@ -45,38 +29,45 @@ String jwt;
 ///////////////////////////////
 // Helpers specific to this board
 ///////////////////////////////
-String getDefaultSensor() {
-  return  "Wifi: " + String(WiFi.RSSI()) + "db";
+String getDefaultSensor()
```

Comment:
Seems like there are a lot of reformatting changes. @gguuss do you know if there's a style that should be followed for consistency?

Reply:
Hey Alex, I switch the format back to the original. I added the `ifdef` stuff to each file so Travis would run quicker on files that shouldn't test certain platforms

Reply:
There is not a standard that we're adhering to, I personally prefer K&R (technically OTBS) but Microsoft (e.g. VSCode) prefers the Allman style so by default VSCode will add newlines around braces.